packages/shorebird_cli/lib/src/shorebird_process.dart

```diff
@@ -1,13 +1,15 @@
 import 'dart:io';
 
 import 'package:meta/meta.dart';
-import 'package:shorebird_cli/src/shorebird_paths.dart';
+import 'package:shorebird_cli/src/shorebird_environment.dart';
 
 typedef RunProcess = Future<ProcessResult> Function(
   String executable,
   List<String> arguments, {
   bool runInShell,
+  Map<String, String>? environment,
   String? workingDirectory,
+  bool resolveExecutables,
```

Comment:
I think you want this to say "useVendedFlutter" or similar.  Since it controls *both* resolving *and* environment.

Reply:
Done